Makefile

```diff
@@ -40,6 +41,9 @@ test_wasm: test_wasm_unit test_wasm_node
 test_ledger: link_wasm
 	cd examples/wasm_ledger && yarn install && yarn test:ledger
 
+test_ledger_legacy: link_wasm
```

Comment:
Do we still need ledger and ledger-legacy? Would it make sense to unify this now?

Reply:
We don't need this anymore. Good catch.